pagopa/io-app-design-system
# [IOAPPX-429] Enable `allowFontScaling` by default in the `IOText` component

## Short description
This PR enables `allowFontScaling` by default in the `IOText` component. Previously the default was tied to the state of the `isExperimental` value, so it was only enabled if you enabled the experimental DS.

## List of changes proposed in this pull request
- Set `allowFontScaling` fallback value to `true` in the `IOText` component 

## How to test
Set larger text on your device and check that the text is dynamic in both standard and experimental views.
